compiler/back_end/cpp/header_generator.py

```diff
@@ -374,7 +374,7 @@ def _get_cpp_view_type_for_physical_type(
             element_view_parameter_types="".join(
                 ", " + p for p in element_view_parameter_types),
             element_size=element_size,
-            addressable_unit_size=parent_addressable_unit,
+            addressable_unit_size=int(parent_addressable_unit),
```

Comment:
Switched this to explicitly convert to `int` to be clear about what we're doing.